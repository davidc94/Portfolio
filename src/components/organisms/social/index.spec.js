import React from 'react';
import renderer from 'react-test-renderer';
import Social from './index';

describe('Social component', () => {
  it('render Social as expected', () => {
    const render = renderer.create(
      <Social />
    );
    expect(render.toJSON()).toMatchSnapshot();
  });
});