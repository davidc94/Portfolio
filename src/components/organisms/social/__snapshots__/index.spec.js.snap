// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Social component render Social as expected 1`] = `
<div
  className="sc-bwzfXH hweCyD"
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "flexDirection": "column",
      "justifyContent": "flex-start",
    }
  }
  width="full"
>
  <div
    className="sc-bwzfXH hweCyD"
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexDirection": "row",
        "justifyContent": "flex-start",
      }
    }
    width="full"
  >
    <div
      className="sc-bwzfXH cvQdJg"
      style={
        Object {
          "flex": 1,
        }
      }
    />
    <div
      className="sc-bwzfXH cvQdJg"
      style={
        Object {
          "flex": 2,
        }
      }
    >
      <p
        className="sc-bdVaJa iLKJWs"
        color="#2B303A"
        size="large"
      >
        Social Stuff
      </p>
    </div>
    <div
      className="sc-bwzfXH cvQdJg"
      style={
        Object {
          "flex": 1,
        }
      }
    />
  </div>
  <div
    className="sc-bwzfXH cvQdJg"
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "justifyContent": "center",
      }
    }
  >
    <a
      href="https://github.com/davidc94"
      target="_blank"
    >
      <div
        className="sc-bwzfXH lfTWUW"
        style={
          Object {
            "margin": "8px",
          }
        }
      >
        <img
          className="sc-htpNat daoksz"
          src=""
        />
      </div>
    </a>
    <a
      href="https://www.linkedin.com/in/davidchellapah/"
      target="_blank"
    >
      <div
        className="sc-bwzfXH lfTWUW"
      >
        <img
          className="sc-htpNat daoksz"
          src=""
        />
      </div>
    </a>
  </div>
</div>
`;
